### ISSUE TITLE
API error: “Class 'verbb\formie\integrations\crm\App' not found

### ISSUE THREAD

**Issue by pbmills** [verbb/formie#1062]
### Describe the bug

I'm trying to use Hubspot integration but keeps giving me API error. Here is the attached screenshot.
<img width="2075" alt="Screen Shot 2022-08-28 at 10 42 53 AM 1" src="https://user-images.githubusercontent.com/31907075/187050497-778e3a44-778c-4ab6-af99-8b79f1aed670.png">

### Steps to reproduce

I'm using the Steps mentioned in the Hubspot integration to make it work. 

### Form settings

- Multi-page form: No
- Submission Method: Ajax

### Craft CMS version

3.7.53.1

### Plugin version

1.6.10

### Multi-site?

No

### Additional context

The CMS is currently working on SOLO.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by pbmills** [comment]
Thanks that worked.

**Comment by pbmills** [comment]
I think the next issue is happening with triggering form "hubspot"
<img width="2107" alt="Screen Shot 2022-08-28 at 10 56 32 AM" src="https://user-images.githubusercontent.com/31907075/187050775-1c630396-1255-4d66-bb68-a7eb73aea27f.png">

Is this something related to formie???

**Comment by engram-design** [comment]
Fixed in [1.6.11](https://github.com/verbb/formie/releases/tag/1.6.11)
